### DISCUSSION TITLE
How can I change the background of pix image viewer from grey to black?

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [linuxmint/discussions#575] [Questions]
How can I do that?
![image](https://github.com/user-attachments/assets/6771c780-605e-4827-9dcc-027a3eca5dae)
I want the grey area around the image to be black. I searched it in the settings, but couldn't find.
OS: Arch

**Comment by Jopp-gh** [comment]
In Pix you can't change app colors. 
Open Mint's `settings>themes` instead.  In there, click on the Advanced button, to configure the user interface. 
You should be able to change from a dark grey to black.
Otherwise, try to download other themes (another tab under themes).

Finally, to modify some specific colors check out: https://www.dedoimedo.com/computers/linux-mint-cinnamon-edit-theme.HTML
But it's more advanced.
